Track screen recording (#462)
rudderlabs/rudderstack-docs: stream-sources/rudderstack-sdk-integration-guides/rudderstack-react-native-sdk.mdx

| `property`   | Object | No       | Extra property object to pass along with the `screen` call. |
| `option`     | Object | No       | Extra options passed along with `screen` event.                     |

### Automatic screen recording

You can enable the automatic recording of screen views by passing `recordScreenViews` as `true` while initializing the `rudderClient`. This automatically sends a `screen` call for every screen that a user views. By default, `recordScreenViews` is set to `false`. 

<div class="infoBlock">

> The <code class="inline-code">recordScreenViews</code> parameter records the screen views via the native Android <a href="https://developer.android.com/guide/components/activities/intro-activities" target="_blank">Activities</a> or the iOS <a href="https://developer.apple.com/documentation/uikit/view_controllers" target="_blank">View Controllers</a> only and <strong>not</strong> by the React Native SDK. 
> </div>  
> 
> To track the screen views via the SDK, you can use the following code snippet:

Reviewer: ```suggestion
The <code class="inline-code">recordScreenViews</code> parameter records the screen views of the native Android <a href="https://developer.android.com/guide/components/activities/intro-activities" target="_blank">Activities</a> or the iOS <a href="https://developer.apple.com/documentation/uikit/view_controllers" target="_blank">Views</a> only and <strong>not</strong> by the React Native Views. 
</div>  

To track the screen views of the React Native Screens, you can use the following code snippet:
```
